.github/workflows/ci.yaml

```
@@ -0,0 +1,26 @@
+name: Build and upload to PyPI
+on: [push, release]
+
+jobs:
+  build_artifacts:
+    name: Build artifacts
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: 22.9.0-2
+          use-mamba: true
+          environment-file: environment.yml
+          activate-environment: vscode-binder
+      - shell: bash -el {0}
+        run:
+          python -m build
+      - uses: pypa/gh-action-pypi-publish@v1.5.1
+        if: github.event_name == 'release'
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TOKEN }}
```

```suggestion
          password: ${{ secrets.PYPI_RELEASE_TOKEN }}
```
sorry, I picked a name without looking here first